src/Utils/TypeInfo.php

```diff
@@ -349,7 +349,7 @@ private static function getFieldDefinition(Schema $schema, Type $parentType, Fie
      */
     public static function typeFromAST(Schema $schema, Node $inputTypeNode): ?Type
```

Comment:
```suggestion
    public static function typeFromAST(callable $typeLoader, Node $inputTypeNode): ?Type
```

Let's bubble up that change even further and make it throughout.